comm/src/main/scala/coop/rchain/comm/package.scala

```diff
@@ -12,4 +17,20 @@ package object comm {
   }
 
   val CommMetricsSource: Metrics.Source = Metrics.Source(Metrics.BaseSource, "comm")
+
+  def isValidInetAddress(host: String): Boolean =
+    Try(InetAddress.getByName(host))
```

Comment:
I don't think `InetAddress.getByName` is a pure function, so it should be enclosed in `F` and delayed

Reply:
@lukasz-golebiewski done